pkgs/top-level/stage.nix

```diff
@@ -213,6 +213,18 @@ let
       };
     } else throw "i686 Linux package set can only be used with the x86 family.";
 
+    # x86_64-darwin packages for aarch64-darwin users to use with Rosetta for incompatible packages
+    pkgsDarwinIntel = if stdenv.hostPlatform.isDarwin then nixpkgsFun {
```

Comment:
I'm not 100% sure about the name.

Other candidates I had in my head were `pkgsIntelDarwin` and `pkgsx86Darwin`.

Suggestions welcome.